src/main/java/BookBob/entity/Patient.java

```diff
@@ -0,0 +1,91 @@
+package BookBob.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Patient {
+    private String name;
+    private String NRIC;
+    private String diagnosis;
+    private ArrayList<String> medication;
+    private int phoneNumber;
+    private String dateOfBirth;
+    private String homeAddress;
```

Comment:
could we store phoneNumber as a string?

because the user inputs phone number as a string,

if we store phoneNumber initially as int, we will need to convert user input from string to int.

but would it be better to just store phoneNumber as string, so we just compare strings. ?

Reply:
also, storing as phoneNumber as string, will be much better for "find"
tq